igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/client/model/Commit.groovy

```diff
@@ -37,7 +37,24 @@ class Commit {
     @SerializedName("author_name")
     String authorName
 
+    @SerializedName("compare_url")
+    String compareUrl
```

Comment:
Is this a github specific thing? I think this `Commit` is supposed to be agnostic to the git host